src/org/usfirst/frc/team4571/robot/RobotConstants.java

```diff
@@ -19,4 +19,10 @@
 	public static final double DRIVE_TRAIN_PULSES_PER_REVOLUTION = 1440.0;
 	public static final double DRIVE_TRAIN_WHEEL_CIRCUMFERENCE = DRIVE_TRAIN_WHEEL_DIAMETER_IN_FEET * Math.PI;
 	public static final double DRIVE_TRAIN_DISTANCE_PER_PULSE = DRIVE_TRAIN_WHEEL_CIRCUMFERENCE / DRIVE_TRAIN_PULSES_PER_REVOLUTION;
+
+    public static final int ROLLER_SOLENOID = 1;
+    public static final int ROLLER_FOWARD_SOLENOID_CHANNEL = 1;
+    public static final int ROLLER_REVERSE_SOLENOID_CHANNEL = 2;
+
+    public static final int RIGHT_MOTOR_CHANNEL = 5;

```

Comment:
May we change the naming of RIGHT_MOTOR_CHANNEL unless we are going to have a LEFT_MOTOR_CHANNEL in the future as well? Even if that is the case, the naming of this will be confusing with our drive motor channels.